### ISSUE TITLE
Labelling subplots

### ISSUE THREAD

**Issue by zeyalt** [ITSLeeds/TDS#37]
Hello Robin, 

You mentioned that one option to create subplots (e.g. 2 plots side by side) was to use "fig,show="hold"" in the R chunk. Something like this:

````{r display_figure_7, eval=TRUE, echo=FALSE, include=TRUE, fig.show="hold", fig.cap="Geospatial distribution of accidents involving pedestrians (left) and cyclists (right) by LSOAs", out.width="47.5%"}

include_graphics("Figures/figure_7a.png")
include_graphics("Figures/figure_7b.png")

```
`

Do you know how we can label each subplot if we use this method? It would be nice if I can give separate labels for the individual subplots as "Figure 7a: Figure showing ..." and "Figure 7b: Figure showing ...". If it's not possible, are there other ways?

**Comment by Robinlovelace** [comment]
> Do you know how we can label each subplot if we use this method?

If you use LaTeX you can do something like this (from @agila5 ): 

```tex
\begin{figure}
  \centering
  \subfloat[][Shortest and fastest paths according to a bicycle weighting profile. \label{figures:bicycle_fastest}]{
    \includegraphics[width=.45\linewidth]{images/bicycle_fastest}
  }
  \hspace{0.05\linewidth}
  \subfloat[][Shortest and fastest paths according to a motorcar weighting profile. \label{figures:motorcar_fastest}]{
    \includegraphics[width=.45\linewidth]{images/motorcar_fastest}
  }
  \caption{Comparison of shortest and fastest routes between Monk Bridge and Chapeltown (Leeds), estimated by \texttt{stplanr} and \texttt{dodgr} algorithms according to two different weighting profiles: bicycle (on the left) and motorcar (on the right). The \texttt{route\_local()} path is coloured in dark-red while the \texttt{dodgr\_paths()} path is coloured in dark-green.}
  \label{figure:shortest_and_fastest_paths}
\end{figure}
```

That works in a .Rmd file if you compile to PDF. I do not know a way of doing it in 'pure' RMarkdown that will also work in .html. Any ideas @agila5 (sorry for nudging you on this and no worries if not)?

A simple solution: just say

    ```{r plotname, fig.cap="Figure showing a (left) and b (right)", ...}

Sound reasonable?

**Comment by zeyalt** [comment]
Thanks for the prompt reply! I thought of doing that but that means I will keep having to say "... In the left graph of Figure 3" or "... as shown in the right graph of Figure 3." 

Anyway, I found the solution in https://stackoverflow.com/questions/55084140/include-figure-labels-in-r-markdown-for-side-by-side-plots. I just needed to add `\usepackage{subfig}` in the YAML header, and `fig.subcap=c("Accidents", "Casualties")` in the R chunk header and I can now refer to specific subplots by labels. 

Thanks anyway!
